src/integration-test/java/org/mskcc/limsrest/service/integrationtest/QcStatusAwareProcessAssignerTest.java

```diff
@@ -1,164 +0,0 @@
-package org.mskcc.limsrest.service.integrationtest;
```

Comment:
Can we just comment out this test?

Reply:
I removed it, the class only had @Before and no tests